src/routing_membrane.rs

```diff
@@ -1558,57 +1536,99 @@ fn populate_routing_node() -> RoutingMembrane<TestInterface> {
 
 #[test]
     fn call_handle_put() {
-        let put_data: PutData = Random::generate_random();
-        assert_eq!(call_operation(put_data,
-            MessageType::PutData, Arc::new(Mutex::new(Stats::new())),
-            Authority::NaeManager(Random::generate_random()), None, None).call_count, 1usize);
+        let mut array = [0u8; 64];
+        thread_rng().fill_bytes(&mut array);
+        let put_data = MessageType::PutData(
+            Data::ImmutableData(
+                ImmutableData::new(ImmutableDataType::Normal, array.iter().map(|&x|x).collect::<Vec<_>>())));
+        assert_eq!(call_operation(put_data, Arc::new(Mutex::new(Stats::new())),
+                   SourceAddress::Direct(Random::generate_random()),
+                   DestinationAddress::Direct(Random::generate_random()),
+                   Authority::NaeManager(Random::generate_random())).call_count, 1usize);
     }
 
 #[test]
 #[ignore]
     fn call_handle_authorised_put() {
-        let unauthorised_put: PutData = Random::generate_random();
-        let result_stats = call_operation(unauthorised_put, MessageType::UnauthorisedPut,
-             Arc::new(Mutex::new(Stats::new())), Authority::Unknown, None, None);
+        let mut array = [0u8; 64];
+        thread_rng().fill_bytes(&mut array);
+        let put_data = MessageType::PutData(
+            Data::ImmutableData(
+                ImmutableData::new(ImmutableDataType::Normal, array.iter().map(|&x|x).collect::<Vec<_>>())));
+        let result_stats = call_operation(put_data, Arc::new(Mutex::new(Stats::new())),
+                   SourceAddress::Direct(Random::generate_random()),
+                   DestinationAddress::Direct(Random::generate_random()),
+                   Authority::Unknown);
         assert_eq!(result_stats.call_count, 1usize);
         assert_eq!(result_stats.data, "UnauthorisedPut".to_string().into_bytes());
     }
 
 #[test]
     fn call_handle_put_response() {
-        let put_data_response: PutDataResponse = Random::generate_random();
-        assert_eq!(call_operation(put_data_response, MessageType::PutDataResponse,
-             Arc::new(Mutex::new(Stats::new())), Authority::NaeManager(Random::generate_random()), None, None).call_count, 1usize);
```

Comment:
prefer using #[ignore] in the future, keeps better track of commented-out tests cheers